File: Interface/frutNatura_funci2/app/src/main/java/com/frutnatura/app/api/MensagemDto.java
package com.frutnatura.app.api;

import com.google.gson.annotations.SerializedName;

public class MensagemDto {
    @SerializedName("id")
    public String id;

    @SerializedName("autorId")
    public String autorId;

    @SerializedName("texto")
    public String texto;

    @SerializedName("criadoEmUtc")
    public String criadoEmUtc;
}

File: Core/FrutNatura.Core.Contracts/Mensagens/MensagemDto.cs
namespace FrutNatura.Core.Contracts.Mensagens;

public sealed class MensagemDto
{
    public Guid Id { get; init; }
    public Guid ChamadoId { get; init; }
    public Guid? AutorId { get; init; }
    public string Texto { get; init; } = default!;
    public DateTime CriadoEmUtc { get; init; }
}
